src/mapbox.common.ts

```diff
@@ -35,16 +35,8 @@ export interface LatLng {
 // ------------------------------------------------------------
 
 export interface QueryRenderedFeaturesOptions {
-    point: LatLng;
-    layerIds?: string[];
-}
-
-// ------------------------------------------------------------
-
-export interface Feature {
-    id: any;
-    type?: string;
-    properties: Object;
```

Comment:
i think for now we should not remove the Feature interface to not introduce a breaking change.
but should be noted and removed in next major release